demo/midgard-node/src/database/utils/user-events.ts

```diff
@@ -124,3 +136,90 @@ export const delEntries = (
       Columns.ID,
     )} IN ${sql.in(ids)}`;
   }).pipe(sqlErrorToDatabaseError(tableName, "Failed to delete given UTxOs"));
+
+export class UserEventsConversionError extends EffectData.TaggedError(
+  "UserEventsConversionError",
+)<SDK.GenericErrorFields> {}
+
+export const makeTransactionUnspentOutput = (
```

Comment:
This helper is only applicable to deposit events, so it shouldn't be defined here.
```suggestion
export const depositEventToCmlTransactionUnspentOutput = (
```